content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst

```diff
@@ -1,64 +1,112 @@
+:guilabel:`Receive Products`, :guilabel:`Validate`, and set :guilabel:`Apply` to set the
+:guilabel:`Done` quantities. Navigate back to the purchase order (via the breadcrumbs).
 
-If this product is always a landed cost, you can also define it on the
-product and avoid having to tick the box on each vendor bill.
+Create a Vendor Bill
+====================
 
-.. image:: integrating_landed_costs/integrating_landed_costs_03.png
-    :align: center
+Now that goods have been received from the purchase order, a vendor bill can be created. To do so,
+click :guilabel:`Create Bill.` The bill will automatically populate with the information from the
+purchase order. To add a landed cost to the bill, click :guilabel:`Edit`.
 
-At the top of my vendor bill, I’ll see a button *create landed costs*.
-I click on this button and a landed cost is automatically created. I can
-now decide on which picking those additional costs should apply.
+While in :guilabel:`Edit` mode, click :guilabel:`Add a line` to add the landed cost product created
+previously. Since this product was created as a landed cost, the checkbox under the
+:guilabel:`Landed Costs` column will be selected by default.
 
-.. image:: integrating_landed_costs/integrating_landed_costs_04.png
-    :align: center
+.. image:: integrating_landed_costs/integrating-landed-costs-vendor-bill.png
+   :align: center
+   :alt: Landed cost added on product lines on vendor bill.
 
-I can now click on *Compute* and go in the tab *Valuation
-Adjustments* to see the impact on my products costs. The last step is
-to validate the landed cost.
+.. important::
+   Since no :guilabel:`Price` was set on the product form for the landed cost, it must be set
+   manually on the vendor bill.
 
-I can access the journal entry that has been created by the landed cost
-by clicking on the journal entry.
+Create Landed Costs
+===================
 
-.. image:: integrating_landed_costs/integrating_landed_costs_05.png
-    :align: center
+Once all the information on the bill has been filled out, click :guilabel:`Create Landed Costs`.
+This will navigate to a new accounting entry for the *Landed Cost*. From here, next to
+:guilabel:`Transfers`, the warehouse receipt with which this landed cost is associated must be
+specified. Additionally, an expenses :guilabel:`Account` must be selected, as well.
+
+To see how the landed costs affect the items on the purchase order, click the :guilabel:`Valuation
+Adjustments` tab. From here, the :guilabel:`Original Value` and :guilabel:`New Value` can be seen,
+as well as the :guilabel:`Additional Landed Cost` that changed the original purchase order.
+
+.. image:: integrating_landed_costs/integrating-landed-costs-valuation-adjustments.png
+   :align: center
+   :alt: Valuation adjustments tab for landed costs on vendor bill.
+
+Once everything is set, :guilabel:`Validate` and :guilabel:`Save`. Then, click back to the
+vendor bill (via the breadcrumbs).
```

Comment:
When I attempted to click 'Validate' I got the following error message: You cannot apply landed costs on the chosen Transfers(s). Landed costs can only be applied for products with FIFO or average costing method.

However, I made sure that FIFO was applied to the Product Category that was applied to BOTH: the product initially added to the RfQ & to the Service Type 'Landed Cost' product added during the Vendor Bill Draft page section. So, perhaps this is a bug? Did I misunderstand a step along the way? Worth looking into either way.